### ISSUE TITLE
Help Wanted: Create AppImages for linux

### ISSUE THREAD

**Issue by fcjr** [ghostery/user-agent-desktop#436]
We'd love some community support to help us support some of the common linux distribution channels.

This ticket is for supporting `AppImages`.

If you are interested in helping us support this distributable format please let us know!

**Comment by 2br-2b** [comment]
I don't know if I'll have the time to maintain this, but I decided to see how easy this would be.

I followed [DistroTube's tutorial](https://odysee.com/@DistroTube:2/create-portable-packages-with-appimage:c?), and successfully created a package for Ghostery. I created this with the second-most-recent release to see if updates would work.

The problem came up when I launched the appimage. It seems to function fine, but it gave me this error:

![image](https://user-images.githubusercontent.com/43968991/125500928-ffca560c-fe7f-4e2e-b34c-ee6b791006c1.png)

I believe that appimages have their own way of updating via AppImageUpdate, so we'd either have to:
1. have to disable the normal update code to set up AppImageUpdate,
2. figure out how to update the AppImage via the built-in Ghostery update process, or
3. disable updating altogether.

TLDR: Packaging the program is simple, updating it is harder
